### PR TITLE
[WIP] add instructions to README.md

### PR DESCRIPTION
**Pull request by thatfiredev**

### CODE REVIEW

**Comment by jamesdaniels** [comment]
Thanks for hacking on this, FYI the tests are a little flaky right now. Namely Node 14 on macos, so ignore, I disabled the branch protection policy for now.

File: README.md [hunk 1]
```diff
@@ -23,6 +29,97 @@
...
+1. Run `firebase serve` from the terminal. This should build your Next.js app and serve it using the Firebase CLI.
+2. Open your web app at the local URL returned by the CLI (usually http://localhost:5000)
+
+### Deploy your app to Firebase Hosting
+
+When you're ready to share your changes with the world, deploy your Next.js app to your live site:
+1. Run `firebase deploy` from the terminal.
+2. Check your website on: `SITE_ID.web.app` or `PROJECT_ID.web.app` (or your custom domain, if you did setup one)
+
+# Integrate Nuxt
```
Comment (davideast) [inline README.md:78]: Switch with Angular, Nuxt is still a WIP for now.

```suggestion
# Deploy Angular
```

File: README.md [hunk 2]
```diff
@@ -23,6 +29,97 @@
 
 This repository is maintained by Googlers but is not a supported Firebase product. Issues here are answered by maintainers and other community members on GitHub on a best-effort basis.
 
+# Integrate Next.js
```
Comment (davideast) [inline README.md:32]:
```suggestion
# Deploy Next.js
```

**Review by davideast — APPROVED** [review]
LGTM with the minor changes.

File: README.md [hunk 3]
```diff
@@ -23,6 +29,97 @@
...
+2. Check your website on: `SITE_ID.web.app` or `PROJECT_ID.web.app` (or your custom domain, if you did setup one)
+
+# Deploy Angular
+
+Easily deploy your Angular application to Firebase and serve dynamic content to your users.
+
+## What you'll need before you begin
+
+### Prerequisites
+- Firebase CLI version 10.8 or later (see installation instructions [here](https://firebase.google.com/docs/cli))
```
Comment (davideast) [inline README.md:85]: `10.9.1` is required now.

Comment (davideast) [inline README.md:85]:
```suggestion
- Firebase CLI version 10.9.1 or later (see installation instructions [here](https://firebase.google.com/docs/cli))
```

File: README.md [hunk 4]
```diff
@@ -23,6 +29,97 @@
...
 This repository is maintained by Googlers but is not a supported Firebase product. Issues here are answered by maintainers and other community members on GitHub on a best-effort basis.
 
+# Deploy Next.js
+
+Easily deploy your Next.js application to Firebase and serve dynamic content to your users.
+
+## What you'll need before you begin
+
+### Prerequisites
+- Firebase CLI version 10.8 or later (see installation instructions [here](https://firebase.google.com/docs/cli))
```
Comment (davideast) [inline README.md:39]:
```suggestion
- Firebase CLI version 10.9.1 or later (see installation instructions [here](https://firebase.google.com/docs/cli))
```

**Comment by thatfiredev** [comment]
@jamesdaniels this still requires a maintainer to approve before it can be merged. Can you take a look and merge?

**Comment by morganchen12** [comment]
Skipping tests and review and merging since it's only README changes per David's request.

**Pull request merged**